intelmq/bin/intelmqcli_create_reports.py

```diff
@@ -20,15 +20,8 @@
 import time
 import zipfile
 
-import psycopg2
```

Comment:
> not needed anymore?

Not in this file. It was only needed to set the extensions which is not necessary anymore as unicode is default.